aea/helpers/transaction/base.py

```diff
@@ -262,6 +332,7 @@ def encode(
         signed_transaction_dict = {
             "ledger_id": signed_transaction_object.ledger_id,
             "body": signed_transaction_object.body,
+            "kwargs": signed_transaction_object.kwargs.body,
```

Comment:
I don't understand why this kwargs is needed here.

Reply:
Okay, so we're not sure if this is required here but ardian thinks adding this here will fix the issue his facing, this needs to be tested. But since this is not blocking testing this is not a big priority

Reply:
It's not in the correct place, needs to be on the performative level. Like [here](https://github.com/valory-xyz/open-aea/tree/main/packages/valory/protocols/ledger_api#:~:text=send_signed_transactions%3A%0A%20%20%20%20signed_transactions%3A%20ct%3ASignedTransactions%0A%20%20%20%20kwargs%3A%20ct%3AKwargs)